samples/batch/js/node/main.js

```diff
@@ -0,0 +1,236 @@
+const { TranscriptionProperties } = require('speech_to_text_api_v30');
+var SpeechToTextApiV30 = require('speech_to_text_api_v30');
+var sleep = require('system-sleep');
+var defaultClient = SpeechToTextApiV30.ApiClient.instance;
+const request = require('request');
+
+// Create API Instance
+var apiInstance = new SpeechToTextApiV30.DefaultApi();
+// Your subscription key and region for the speech service
+
+const API_KEY=""
+const NAME = "Simple transcription"
+const DESCRIPTION = "Simple transcription description"
+const LOCALE = "en-US"
+
+// Provide the uri of the audio file 
+const RECORDINGS_BLOB_URI = ""
+
+// Provide the uri of a container with audio files for transcribing all of them with a single request
+const RECORDINGS_CONTAINER_URI = ""
+
+var callbackGetTrans = function(error, data, response) {
+  if (error) {
+    console.error(error);
+  } else {
+    console.log('Callback Get Transcription API called successfully. Returned data: ' + JSON.stringify(data));
+    completed=false ;
+    var tid = data.self.substring(data.self.lastIndexOf("/")+1);
+    console.log("Transcription Id " + tid);
+    status = _checkComplete(tid);
+
+    if (status == "Succeeded")
+    {
+        var opts = _getTranscriptionFiles();
+        apiInstance.getTranscriptionFiles(tid,opts,callbackGetTransFiles);
+    }
+    else if(data.status == "Failed")
+           console.log("Transcription failed: {transcription.properties.error.message}")
+  }
+};
+
+var callbackGetTransFiles = function(error, data, response) {
+  if (error) {
+    console.error(error);
+  } else {
+   console.log('callbackGetTransFiles API called successfully. Returned data: ' + JSON.stringify(data));
+   pag_files = data.values;
+   for (var i = 0; i < pag_files.length; i++)
+   {
+     var file_data = pag_files[i];
+     var url = file_data.links.contentUrl; 
+     if(file_data.kind !='Transcription')
+        continue;
+     else{
```

Comment:
nit: please use consistent indentation and spacing before/after brackets and operators.